### PR TITLE
Add beer and sing option to life puzzle story

### PR DESCRIPTION
**Pull request by PiotrBerebecki**
Please review this, @casey-collab

### CODE REVIEW

File: english/LifePuzzle/LifePuzzle.md [hunk 1]
```diff
@@ -5,4 +5,11 @@ and your mass is proportionally reduced
 so as to maintain your original density. 
 You are then thrown into an empty glass blender. 
 The blades will start moving in 60 seconds. 
-What do you do?
\ No newline at end of file
+What do you do?
+
+Drink some [bear](bear/bear.md)
```
Comment (casey-collab) [inline english/LifePuzzle/LifePuzzle.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/LifePuzzle/LifePuzzle.md [hunk 2]
```diff
@@ -5,4 +5,11 @@ and your mass is proportionally reduced
...
+
+or
+
+Begin to [sing](sing/sing.md)
```
Comment (casey-collab) [inline english/LifePuzzle/LifePuzzle.md]: This link is also invalid.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by PiotrBerebecki** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :dog:

**Pull request merged**